### DISCUSSION TITLE
Welcome to av CLI Discussions!

### DISCUSSION THREAD

**Discussion by jainankit** [aviator-co/av#189] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with others who are using the `av` CLI. We hope that you:
  * Ask questions about the CLI
  * Share ideas or any feature requests. You can also create issue tickets if you need more concrete answer.
  * Help answer questions for other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about how you use av CLI.

**Comment by jainankit** [comment]
Hey everyone, I'm the cofounder of Aviator. Aviator is a suite of productivity tools purpose-built to reduce friction in development workflows.

This CLI is an open-source CLI built and managed by Aviator to interact with GitHub and Aviator service. The primary use case is creating and managing Stacked PRs. Would love to hear any feedback you have on the CLI.

**Comment by ndrone-kr** [comment]
Nice! question though, why is this a separate install, and not part of the GitHub CLI extensions?

**Reply by jainankit** [reply]
No particular reason, we could consider adding this as part of the GitHub CLI extensions in the future.
